README.md

```diff
@@ -24,43 +24,44 @@ See [Testing Farm docs](https://docs.testing-farm.io) for more information on su
 
```

Comment:
I suggest leaving the tables formatted as they were before.
That way is easier to detect git diffs.

Reply:
Resolved.